ui/scss/component/_card.scss

```diff
@@ -63,7 +65,19 @@ $padding-card-horizontal: $spacing-vertical * 2/3;
   white-space: pre-wrap;
 }
 .card__subtext--two-lines {
-  height: $font-size * 0.9 * $font-line-height * 2;
+  height: ($font-size * 0.9 * $font-line-height + 6) * 2;
```

Comment:
"+6" makes this magic CSS, if this 6 references something else, it should be a variable, if it was determined by trial and error, it's a hack